README.md

```diff
@@ -1,2 +1,76 @@
 # ra-auth-google
-Auth provider for react-admin using the Google Identity Services
+
+An auth provider for [react-admin](https://github.com/marmelab/react-admin) that handles authentication using the [Google Identity Services (GIS)](https://developers.google.com/identity/gsi/web/guides/overview?hl=en).
+
+It allows to easily enable users to sign to your app in using their Google account, either personal, or professional via [Google Workspaces](https://workspace.google.com/).
+
+This repository contains:
+
+-   The actual `ra-auth-google` package
+-   A simple demo app you can run locally to try out `ra-auth-google` with your own Google API Client ID
+
+## The `ra-auth-google` package
+
+-   Please have a look at the [DOCUMENTATION](./packages/ra-auth-google/Readme.md)
+-   And also the [source code](https://github.com/marmelab/ra-auth-google/tree/main/packages/ra-auth-google)
+
+## Simple Demo
+
+### Prerequesites
+
+To enable Sign In With Google on your website, you first need to set up your **Google API client ID**.
+
+Please follow this [setup guide](https://developers.google.com/identity/gsi/web/guides/get-google-api-clientid?hl=en) to get your Google API client ID and configure your OAuth Consent Screen.
+
+If you choose to create a new Google APIs project in Test mode, feel free to add some **test users**.
+
+When prompted for the **Authorized JavaScript origins**, please add the following values:
+
+- `http://localhost:8080`
+- `http://localhost`
+
+### Initial setup
+
+First, clone this project.
+
+```sh
+git clone https://github.com/marmelab/ra-auth-google.git
+cd ra-auth-google
+```
+
+Then, configure the demo app with your **Google API client ID**. Create a `.env` file in the root of the project.
+
+```sh
+cp packages/demo-react-admin/.env.template packages/demo-react-admin/.env
```

Comment:
This can be added as a makefile command:
```
cp -n packages/demo-react-admin/.env.template packages/demo-react-admin/.env
```

Note the `-n` to avoid overriding an existing file